docs/products/kafka/concepts/schema-registry-authorization.rst

```diff
@@ -0,0 +1,103 @@
+Schema registry authorization
+=============================
+
+Schema registry authorization feature enabled in :doc:`Karapace schema registry <../howto/enable-karapace>`  allows you to both authenticate the user, and additionally grant or deny access to individual `Karapace schema registry REST API endpoints <https://github.com/aiven/karapace>`_ and filters the content the endpoints return.
+
+.. Tip::
+
+  Karapace schema registry authorization has been available in Aiven since 2022-06-30 in all Aiven for Apache Kafka® services created after that date have it enabled by default and it's not possible to disable it.
+
+  For the services created before 2022-06-30 the feature needs to be enabled, check the :doc:`dedicated article <../howto/enable-schema-registry-authorization>` to understand how to enable the feature.
+
+.. _karapace_schema_registry_acls:
+
+Karapace schema registry ACLs
+-----------------------------
+
+Karapace schema registry authorization is configured using dedicated Access Control List (ACL). 
+
+Karapace schema registry ACL definition
+'''''''''''''''''''''''''''''''''''''''
+
+A Karapace schema registry authorization ACL consists of zero or more entries that each specify a **username**, an **operation** and a **resource**.
+
+* The **username** is the name of a service user in the Aiven for Apache Kafka service.
+* The **operations** are: 
+  
+  * ``schema_registry_read`` (later: ``read``)
+  * ``schema_registry_write`` (later: ``write``). ``write`` implies ``read``.
+
+* The **resource** is can be in the following formats: 
+
+  * ``Config:``: the entry controls the access to global compatibility configurations. As of Karapace allows only to retrieve and change the default schema compatibility mode via the global ``Config:`` resource, check the `project README for more information <https://github.com/aiven/karapace/blob/main/README.rst>`_.
+
+    .. Note::
+
+      The global compatibility APIs require ``Config:`` resource access with ``Read`` permission when getting the configuration and ``Write`` permission when setting it.
+
+  * ``Subject:subject_name``: the entry controls the access to subjects in Schema Registry
+    
+
+.. Tip::
+
+  The ``name`` specified in the ACL entry resource and username can use wildcards:
+      
+  * ``*`` matching any characters
+  * ``?`` matching a single character
+
+The algorithm to determine if a request has the necessary permissions is the following:
+
+* Iterate over ACL entries in the order
+* Check if the current requesting user matches the ACL entry and the resource in the ACL entry matches the accessed resource
+* If it matches, use that ACL entry. Check whether the operation in the ACL entry matches the accessed operation, based on that grant or deny access
+* If no ACL entry is found, deny access, with HTTP response having status code 401
```

Comment:
This is not how this has been implemented. The implementation grants access if any of ACL entries matches, and allows access. The order does not matter.